### PR TITLE
Adding support for galaxy operator

### PR DESCRIPTION
**Pull request by aknochow**
1. Updating Dockerfile to install required directories in /var/lib/pulp/ 
2. Adding skopeo dnf dependency for content signing script 
3. Adding readyz.py for galaxy-operator ready check

depends-on: https://github.com/ansible/galaxy-operator/pull/17

'No-Issue'

### CODE REVIEW

**Review by rooftopcellist — CHANGES_REQUESTED** [review]

File: galaxy-operator/bin/wait_on_postgres.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
```
[the remark this answers is not in the document]
Comment (aknochow) [reply galaxy-operator/bin/wait_on_postgres.py]: done!

**Review by drodowic — APPROVED** [review]

File: galaxy-operator/bin/readyz.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    print("Readiness probe: ready!")
+    sys.exit(0)
+
+
+"""
+PID 1 value (/proc/1/cmdline) may vary based on the gunicorn install method (RPM vs pip)
+The cmdline value for this PID looks like:
+```
+# pip installation
+gunicorn: master \[pulp-{content,api}\]
```
Comment (rooftopcellist) [inline galaxy-operator/bin/readyz.py:51]: The CI linter is just complaining about the backslashes here in the docstring.

```
Run [ ! -x .ci/scripts/extra_linting.sh ] || .ci/scripts/extra_linting.sh
./galaxy-operator/bin/readyz.py:51:18: W605 invalid escape sequence '\['
./galaxy-operator/bin/readyz.py:51:38: W605 invalid escape sequence '\]'
```

We can just remove them imo.

**Review by rooftopcellist — APPROVED** [review]
This changeset looks good.  

Idea: I think we might want to think about adding a "configure-signing-services.sh" for the initContainer to use in a future PR to make the operator codebase a bit cleaner.  

But a PR for another day.

**Comment by rooftopcellist** [comment]
@himdel @drodowic @jctanner It looks all of the CI issues pertaining to this PR have been fixed.  But the Galaxy CI checks are all failing due to an unrelated problem now when building the image:

```
    7.066 ERROR: Package 'galaxy-ng' requires a different Python: 3.8.17 not in '>=3.9'
    ------
    Containerfile:8
    --------------------
       6 |     # Install python packages
       7 |
       8 | >>> RUN pip3 install ./galaxy_ng
```

I see this in other PR's as well for example:
* https://github.com/ansible/galaxy_ng/pull/2066

Actually, it looks like this might just need a rebase to pick up this change:
* https://github.com/ansible/galaxy_ng/pull/2048/files#diff-a0aba5a2bd30b2f80a3a18aa24ce43670b1320ca0a8b9c103bb3bc971ae30a85R33

We'll try that, cc @aknochow

**Comment by rooftopcellist** [comment]
It appears that the base image used by CI when building has python3.8.17, and python3.9+ is required.  

```
[chadams@awx galaxy_ng]$ docker run -it ghcr.io/pulp/pulp-ci-centos bash

[root@2bfd2acc2a67 /]# python3 --version
Python 3.8.17
```

**Pull request merged**